### PR TITLE
 New Wind, Solar, Geothermal and Biomass for Portugal

### PR DESCRIPTION
**Pull request by Falconet8**
Based on this official report - https://www.dgeg.gov.pt/media/1tbduhyk/dgeg-arr-2021-12.pdf
New values for the year of 2021.

### CODE REVIEW

**Comment by Macquaria** [comment]
Hi, 

In Portugal, installed capacity for geothermal power should be 0 MW instead of 34 MW. As mentioned page 9 of the report posted by @Falconet8, all of the country's geothermal capacity is located in Azores, which are not part of the same grid.

For similar reasons, all capacity from Azores and Madeira should be removed from Electricitymap's Portugal figures.

Wind : -98 MW 
Biomass : -13 MW
Solar : -22 MW

File: config/zones.json [hunk 1]
```diff
@@ -5103,16 +5103,16 @@
       ]
     ],
     "capacity": {
-      "biomass": 684,
+      "biomass": 892,
       "coal": 0,
       "gas": 4580,
-      "geothermal": 29,
+      "geothermal": 34,
```
Comment (Macquaria) [inline config/zones.json:5109]: Should be 0

File: config/zones.json [hunk 2]
```diff
@@ -5103,16 +5103,16 @@
...
       "hydro": 4372,
       "hydro storage": 2827,
       "nuclear": 0,
       "oil": 0,
-      "solar": 861,
-      "wind": 5259
+      "solar": 1386,
```
Comment (Macquaria) [inline config/zones.json:5114]: Should be 1364

File: config/zones.json [hunk 3]
```diff
@@ -5103,16 +5103,16 @@
...
       "coal": 0,
       "gas": 4580,
-      "geothermal": 29,
+      "geothermal": 34,
       "hydro": 4372,
       "hydro storage": 2827,
       "nuclear": 0,
       "oil": 0,
-      "solar": 861,
-      "wind": 5259
```
Comment (Macquaria) [inline config/zones.json:5115]: Should be 5161

File: config/zones.json [hunk 4]
```diff
@@ -5103,16 +5103,16 @@
       ]
     ],
     "capacity": {
-      "biomass": 684,
+      "biomass": 892,
```
Comment (Macquaria) [inline config/zones.json:5106]: Should be 879

**Comment by Falconet8** [comment]
Per @Macquaria remark, I have updated the Wind, Biomass, Solar and Geothermal values. Geothermal has been removed since it is only available on the Azores while the remaining values have been updated to reflect only the installed capacities on Continental Portugal.

**Comment by madsnedergaard** [comment]
Awesome, and great catch @Macquaria! Minor comment, but the numbers suggested by @Macquaria are slightly different from the ones used - any reason why? I briefly looked at the report, but my Spanish is not useful enough to figure out what is the true values :D

**Comment by Falconet8** [comment]
> 

Hi, I used the capacity numbers for each of the energy sources and then subtracted the Azores and Madeira figures.
Wind: Page 15, on the bottom table where it says "Continente" and "2021" - The figure is 5528.
Biomass: Page 17, bottom table ,where it says "Continente" and "2021" - The figure is 876.
Solar: Page 18, on the table called "Potência Instalada por Região (MW)", "Continente", "2021" - The figure is 1,364.

**Review by madsnedergaard — APPROVED** [review]
Sounds good, let's merge this then :)

**Pull request merged**